scripts/distribute_ios.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Normally for Omise, When scripts file is used for buildkite pipeline. We  will store those scripts under `.buikdkite` na krub.  
Example 
![Uploading image.png…]()